Hestia/app/src/main/java/com/rugged/application/hestia/PeripheralLab.java

```diff
@@ -0,0 +1,52 @@
+package com.rugged.application.hestia;
+
+import android.content.Context;
+import android.util.Log;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class PeripheralLab {
```

Comment:
I had the same thing, its probably a good idea to add some comments in these parts.